### PR TITLE
[OCPCLOUD-809] Add verify-diff check in generate task and enable in CI

### PR DESCRIPTION
**Pull request by Danil-Grigorev**
Allowing automatic make generate checks in CI. It allows to keep track of API changes, client-generated code updates on each PR.

This PR is dependent on openshift/release#13400 to enable the functionality in CI.

### CODE REVIEW

**Review by alexander-demichev — APPROVED** [review]
/approve

File: Makefile [hunk 1]
```diff
@@ -105,6 +106,7 @@ fmt: ## Go fmt your code
 .PHONY: goimports
 goimports: ## Go fmt your code
```
Comment (elmiko) [inline Makefile:107]: maybe we should fix the comment here?

**Review by elmiko — COMMENTED** [review]
this is ok Danil, but since we are adjusting the makefile i think we should fix the comment

**Review by elmiko — APPROVED** [review]
thanks Danil!
/lgtm

**Pull request merged**